net-im/skypeforlinux/skypeforlinux-1.9.0.2.ebuild

```diff
@@ -10,7 +10,6 @@ inherit eutils rpm
 DESCRIPTION="P2P Internet Telephony (VoiceIP) client"
 HOMEPAGE="http://www.skype.com/"
 SRC_URI="https://repo.skype.com/rpm/stable/skypeforlinux_${PV}-1.x86_64.rpm"
```

Comment:
I tried doing that and portage barfed.  soemthing about N.N.N.N-N being an invalid version number